deploy/kubernetes/examples/demo-volumesnapshotclass-config-secret.yaml

```diff
@@ -8,10 +8,11 @@ parameters:
   # non-csi.storage.k8s.io parameters may be specified in by_management_id per system and/or outside by_management_id as the cross-system default.
 
   by_management_id: '{"demo-management-id-1":{"pool":"demo-pool-1","SpaceEfficiency":"dedup_compressed","snapshot_name_prefix":"demo-prefix-1"},
-                      "demo-management-id-2":{"pool":"demo-pool-2","snapshot_name_prefix":"demo-prefix-2"}}'  # Optional.
+                      "demo-management-id-2":{"pool":"demo-pool-2","snapshot_name_prefix":"demo-prefix-2","flashcopy_2":"false"}}'  # Optional.
   pool: demo-pool                    # Optional. Use to create the snapshot on a different pool than the source.
   SpaceEfficiency: thin              # Optional. Use to create the snapshot with a different space efficiency than the source.
   snapshot_name_prefix: demo-prefix  # Optional.
+  flashcopy_2: "false"               # Optional. Values true/false. The default is false.
```

Comment:
similar to io/volume groups, I don't think we need to keep adding every new parameter in this cumbersome topology example

Reply:
ok, done

Reply:
same for `demo-storageclass-config-secret.yaml`

Reply:
done